### ISSUE TITLE
Your github page is too cool

### ISSUE THREAD

**Issue by Christophe-Foyer** [peterdsharpe/peterdsharpe#1]
Your github page is too fancy. Mine isn't. What do?

**Comment by peterdsharpe** [comment]
[I see you're well on your way! :)](https://github.com/Christophe-Foyer/Christophe-Foyer)

**Comment by peterdsharpe** [comment]
Happy to send you the photoshop `*.psd` file for the background image I made if you wanna repurpose it! Just a simple image + text + blur filter. The key to getting it to look right is blurring under the text + darkening the image

**Comment by Christophe-Foyer** [comment]
Yeah it's good looking! I think I'll manage to make my own but the blur works really well and I might steal the idea 😉
